src/sctp_transport/mod.rs

```diff
@@ -333,6 +338,51 @@ impl RTCSctpTransport {
         self.state.load(Ordering::SeqCst).into()
     }
 
+    pub(crate) async fn collect_stats(
+        &self,
+        collector: &Arc<Mutex<StatsCollector>>,
+        worker: Worker,
+        peer_connection_id: String,
+    ) {
+        let dtls_transport = self.transport();
+
+        // TODO: should this be collected?
+        dtls_transport
+            .collect_stats(collector, worker.clone())
+            .await;
```

Comment:
I don't see this specific metric being tracked in Pion, but since we're getting `dtls_transport` stats elsewhere it's easy to do.